src/styles/_catalog.scss

```
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 0.5s, color 0.5s;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2);
+    }
+
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
 }
```

According to css style order, animation declarations should go with a separate "block" and emty lines are not needed between parent and son closing tags.
-[css order](https://codeguide.academy/html-css.html#css-order)
```suggestion
    margin-bottom: 48px;
    
    transition: transform 0.5s, color 0.5s;

    &:hover {
      color: #34568b;
      transform: scale(1.2);
    }
  }

  &__card:not(:nth-child(4n)) {
    margin-right: 48px;
  }
}
```